modules/lambdas/main.tf

```diff
@@ -118,6 +118,35 @@ resource "aws_lambda_function" "perform_orca_reconcile" {
   }
 }
 
+# internal_reconcile_report_phantom - Receives job id and page index from end user and returns reporting information of files that have records in the ORCA catalog but are missing from S3 bucket.
+# ==============================================================================
+resource "aws_lambda_function" "internal_reconcile_report_phantom" {
+  ## REQUIRED
+  function_name = "${var.prefix}_internal_reconcile_report_phantom"
+  role          = var.restore_object_role_arn
+
+  ## OPTIONAL
+  description      = "Returns reconcilliation report sample data."
+  filename         = "${path.module}/../../tasks/internal_reconcile_report_phantom/internal_reconcile_report_phantom.zip"
+  handler          = "internal_reconcile_report_phantom.handler"
+  memory_size      = var.orca_reconciliation_lambda_memory_size
+  runtime          = "python3.7"
```

Comment:
I think we want to move over to python 3.9 moving forward

Reply:
Good catch. Fixed.